podcast/the-changelog-384.md

```diff
@@ -76,15 +76,15 @@ Anyway, long story - basically, we've gone to the other extreme. But in the end,
 
 One thing with email is that there was nothing before it, as if you had to have a bootstrap first commonality to let people talk to one another... So whatever the first thing was that could bridge that gap in a global way would have amazing first-mover advantage, as we've seen. Secondly, it's open federation, whereas IRC is closed federation nowadays. In the very dim days of IRC, right at the beginning where there was only one network, it was actually open federation and anybody could just pitch up and try to peer in, or set up an S: line or whatever it is to go and connect to another server... And it was a disaster, because it didn't scale, and it got abused.
 
-Then there was the creation of EFnet, which I think was the first fork out of the original IRC network, when they \[unintelligible 00:14:57.18\] and so on and so forth. But it was basically a closed federation, so you couldn't just jump on board. Probably the other major thing is the lack of a killer app for IRC. When we look at where open standards for communication have happened before, like SIP for VoIP, and XMPP obviously for chat, and IRC; there's never for any of them been a no-brainer killer app like Netscape was for the early web, or like -- I hate to say it, but like Outlook Express was in the early days of email, relatively early days of email. Let's face it, every single Windows box had Outlook Express in there, and people used it...
+Then there was the creation of EFnet, which I think was the first fork out of the original IRC network, when they juped off a server called Jupiter I think, and so on and so forth. But it was basically a closed federation, so you couldn't just jump on board. Probably the other major thing is the lack of a killer app for IRC. When we look at where open standards for communication have happened before, like SIP for VoIP, and XMPP obviously for chat, and IRC; there's never for any of them been a no-brainer killer app like Netscape was for the early web, or like -- I hate to say it, but like Outlook Express was in the early days of email, relatively early days of email. Let's face it, every single Windows box had Outlook Express in there, and people used it...
```

Comment:
This sounds like "jupitered off" to me, though I'm not familiar with either phrase 🤔